### PR TITLE
TLC-70: Obstetric history labels, patient documents fix

### PR DESCRIPTION
**Pull request by ouiliam**
modified:   configuration/concepts/concepts.csv

### CODE REVIEW

File: configuration/drugs/drugs.csv [hunk 1]
```diff
@@ -33,7 +34,7 @@ Uuid,Void/Retire,Name,Concept Drug,Concept Dosage Form,Strength,_version:1
 ,,Doxycycline 100mg Tablet,Cambodia_Doxycycline,Tablet,100mg,
 ,,Erythromycin 500mg Tablet,Cambodia_Erythromycin,Tablet,500mg,
 ,,Erythromycin 250mg Tablet,Cambodia_Erythromycin,Tablet,250mg,
-,,Folic acid 220mg Tablet,Cambodia_Folic acid,Tablet,220mg,
+,,Folic acid 5mg Tablet,Cambodia_Folic acid,Tablet,5mg,
```
Comment (rbuisson) [inline configuration/drugs/drugs.csv:37]: @ouiliam , renaming this drug will create a duplicate when deployed on the production server.
Could you enforce the editing of this drug line by inputting the UUID the existing drug has in production?

File: configuration/drugs/drugs.csv [hunk 2]
```diff
@@ -56,15 +58,18 @@ Uuid,Void/Retire,Name,Concept Drug,Concept Dosage Form,Strength,_version:1
 ,,Paracetamol 200mg Tablet,Cambodia_Paracetamol,Tablet,200mg,
 ,,Paracetamol 325mg Tablet,Cambodia_Paracetamol,Tablet,325mg,
 ,,Paracetamol 100mg Tablet,Cambodia_Paracetamol,Tablet,100mg,
-,,"Penicillin V 1,000,000IU Tablet",Cambodia_Penicillin V,Tablet,"1,000,000IU",
+,,"Penicillin V 1,000,000iu Tablet",Cambodia_Penicillin V,Tablet,"1,000,000iu",
```
Comment (rbuisson) [inline configuration/drugs/drugs.csv:61]: Same thing here. Renaming will create a new drug.
Modifying the FSN means modifying the drug identification key. With the new name, the Initializer will see it as a new drug and create it. Same as for Folic Acid 5mg, just find the UUID it has been assigned in production and input it in the UUID column.

File: configuration/drugs/drugs.csv [hunk 3]
```diff
@@ -56,15 +58,18 @@ Uuid,Void/Retire,Name,Concept Drug,Concept Dosage Form,Strength,_version:1
...
 ,,Praziquantel 600mg Tablet,Cambodia_Praziquantel,Tablet,600mg,
 ,,Prednisolone 5mg Tablet,Cambodia_Prednisolone,Tablet,5mg,
+,,Ranitidine 300mg Tablet,Cambodia_Ranitidine,Tablet,300mg,
 ,,Salbutamole 2mg Tablet,Cambodia_Salbutamole,Tablet,2mg,
 ,,Salbutamole 100mcg/actuation Inhaler,Cambodia_Salbutamole,Inhaler,100mcg/actuation,
+,,Salbutamole 2.5mg/2.5ml Inhaler,Cambodia_Salbutamole,Inhaler,2.5mg/2.5ml,
 ,,Spironolactone 25mg Capsule,Cambodia_Spironolactone,Capsule,25mg,
-,,"Vitamine A 200,000IU Tablet",Cambodia_Vitamine A,Tablet,"200,000IU",
+,,Spironolactone 50mg Capsule,Cambodia_Spironolactone,Capsule,50mg,
+,,"Vitamine A 200,000iu Tablet",Cambodia_Vitamine A,Tablet,"200,000iu",
```
Comment (rbuisson) [inline configuration/drugs/drugs.csv:72]: Same here as well (see remarks above). FSN is renamed. New concept will be created.
Just find the UUID in the prod server and add it to this drug line.

Comment (ouiliam) [reply configuration/drugs/drugs.csv:72]: @rbuisson I changed these 3 to match the changed name already in prod server, so is this necessary?

Comment (rbuisson) [reply configuration/drugs/drugs.csv:72]: @ouiliam that's great if you have changed it on prod already, so then no that is not necessary to do add the UUID.

**Comment by rbuisson** [comment]
Commits are now cherry-picked on 0.90.x.
Closing the PR.

**Pull request closed without merging**